### PR TITLE
PILCS-746 PILCS-750 add reqid

### PR DESCRIPTION
**Pull request by lansefengxinzi**

### CODE REVIEW

File: Pod/Classes/pili-librtmp/rtmp.c [hunk 1]
```diff
@@ -51,6 +52,7 @@ TLS_CTX RTMP_TLS_ctx;
 static const int packetSize[] = {12, 8, 4, 1};
 
 int PILI_RTMP_ctrlC;
+char reqid[30];
```
Comment (longbai) [inline Pod/Classes/pili-librtmp/rtmp.c]: static

File: Pod/Classes/pili-librtmp/rtmp.c [hunk 2]
```diff
@@ -1001,6 +1003,24 @@ int PILI_RTMP_Connect1(PILI_RTMP *r, PILI_RTMPPacket *cp, RTMPError *error) {
 }
 
 int PILI_RTMP_Connect(PILI_RTMP *r, PILI_RTMPPacket *cp, RTMPError *error) {
+    //获取hub
+    char hub[4];
```
Comment (longbai) [inline Pod/Classes/pili-librtmp/rtmp.c]: char hub[5] = {0}

File: Pod/Classes/pili-librtmp/rtmp.c [hunk 3]
```diff
@@ -1001,6 +1003,24 @@ int PILI_RTMP_Connect1(PILI_RTMP *r, PILI_RTMPPacket *cp, RTMPError *error) {
...
+    }
+    
+    if (strlen(hub)>0) {
+        time_t nowtime;
+        time ( &nowtime );
+        char tempTime[20]={0};
+        sprintf(tempTime,"%ld",nowtime);
+        reqid[0] = '\0';
+        strcat(reqid, hub);
+        strcat(reqid, tempTime);
```
Comment (longbai) [inline Pod/Classes/pili-librtmp/rtmp.c]: strncat

File: Pod/Classes/pili-librtmp/rtmp.c [hunk 4]
```diff
@@ -4292,4 +4323,8 @@ int PILI_RTMP_Write(PILI_RTMP *r, const char *buf, int size, RTMPError *error) {
 
 int PILI_RTMP_Version() {
     return MAJOR * 100 * 100 + MINOR * 100 + PATCH;
+}
+
+char * PILI_Get_ReqId(){
```
Comment (longbai) [inline Pod/Classes/pili-librtmp/rtmp.c]: const char*

File: Pod/Classes/pili-librtmp/rtmp.h [hunk 5]
```diff
@@ -352,6 +352,8 @@ int PILI_RTMP_Write(PILI_RTMP *r, const char *buf, int size, RTMPError *error);
 
 int PILI_RTMP_Version();
 
+char * PILI_Get_ReqId();
```
Comment (longbai) [inline Pod/Classes/pili-librtmp/rtmp.h]: PILI_RTMP_Get

File: Pod/Classes/pili-librtmp/rtmp.c [hunk 6]
```diff
@@ -1609,6 +1629,7 @@ SAVC(secureToken);
 SAVC(secureTokenResponse);
 SAVC(type);
 SAVC(nonprivate);
+SAVC(reqid);
```
Comment (longbai) [inline Pod/Classes/pili-librtmp/rtmp.c]: SAVC(X-Reqid)

**Comment by longbai** [comment]
LGTM

**Comment by cnwangsiyu** [comment]
LGTM

**Pull request merged**